tests/spec/smoketest-eq-services-spec.js

```diff
@@ -0,0 +1,49 @@
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+
+chai.use(chaiAsPromised);
+chai.should();
+
+const AuthorHelpers = require('../author-helpers');
+const DesignQuestionnairePage = require('../pages/design-questionnaire.page');
+
+describe('eQ Services Smoke Test', () => {
```

Comment:
I thought we changed these from `() => {` to `function() {`

Reply:
eslint doesn't like it and fixed it when I tried yarn lint --fix